.eslintrc.json

```diff
@@ -30,7 +32,10 @@
     // to deal with false positives in type annotations, turn on typescript-specific rules
     "no-unused-vars": "off",
     "@typescript-eslint/no-unused-vars": ["error", {"varsIgnorePattern": "^_"}],
-    "id-length": "error"
+    "id-length": "error",
+    "import/no-unresolved": "off",
+    "import/no-named-as-default": "off",
```

Comment:
keeps giving warning for `import styled from styled components`
searched online and seems like this particular rule doesn't work well (or accurately)